modules/serverless-logic-editing-workflows.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-editing-workflows"]
+= Editing a workflow
+
+When the {ServerlessLogicOperatorName} deploys a Workflow service, it creates two ConfigMaps to store runtime properties:
```

Comment:
{ServerlessLogicOperatorName} will not reflect right now because I have added this attribute in another (https://github.com/openshift/openshift-docs/pull/76488). They are linked. Once we merge that PR this attribute will reflect correctly.